src/features/weather/components/current_weather.tsx

```diff
@@ -0,0 +1,13 @@
+
+interface CurrentWeatherProps {
+  currentWeather: Record<string, unknown>
+}
+
+export const CurrentWeather = (props: CurrentWeatherProps) => {
+  console.log(props)
+  return (
+    <>
+      {/* <div>Current Weather</div> */}
```

Comment:
Add the template stuff here. Can use `<Stack>` & `<Item>` components to easily create some structure. We are using MUI. The docs are pretty good. https://mui.com/material-ui/react-stack/